applications/mne_analyze/mne_analyze/mainwindow.cpp

```diff
@@ -189,6 +235,47 @@ void MainWindow::createActions()
 
 //=============================================================================================================
 
+void MainWindow::onStyleChanged(const QString& sStyle)
+{
+    qDebug() << "MainWindow::onStyleChanged " << qobject_cast<QApplication *>(QApplication::instance());
+    // Styles are from https://github.com/Alexhuszagh/BreezeStyleSheets
+    if(QApplication *pApp = qobject_cast<QApplication *>(QApplication::instance())) {
+        if(sStyle == "default") {
+            m_sCurrentStyle = "default";
+            pApp->setStyleSheet("MainWindow::onStyleChanged - hello");
+        } else {
+            qDebug() << "hello";
```

Comment:
Code convention/style changes: leftover debug message. Other instances of QDebug could be replaced with QInfo or QWarning